custom/enikshay/management/commands/run_testing_facility_name_migration.py

```diff
@@ -58,17 +58,19 @@ def handle(self, commit, domain, log_path, **options):
             "real" if commit else "fake", domain, datetime.datetime.utcnow()
         ))
 
-        cases = (CaseSearchES()
-                 .domain(domain)
-                 .case_type("test")
-                 .case_property_query("result_recorded", "yes", "must")
-                 .values_list('case_id', flat=True))
+        case_ids = [
+            hit['_id'] for hit in (CaseSearchES()
```

Comment:
You may be able to use `get_ids` or `source` to specify that you only want the ids returned